NEWS.md

```diff
@@ -1,3 +1,14 @@
+## 1.0.3 (Released)
```

Comment:
We discussed with Piotr and decided that since we have started to support the new version of the instance-storage, the logic has changed slightly, as well as the configuration-client and raml-module-builder versions have changed, we should release it under version 1.1.0.